erpnext/patches/v10_0/workflow_leave_application.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc("hr", "doctype", "leave_application")
+	frappe.reload_doc("workflow", "doctype", "workflow")
+
+	doc = frappe.get_doc({
+		'doctype': 'Workflow State',
+		'workflow_state_name': 'Open',
+		'style': 'Warning'
+		}).insert(ignore_permissions=True)
```

Comment:
File "/Users/rohitwaghchaure/Documents/erpnext/frappe-bench/apps/frappe/frappe/modules/patch_handler.py", line 83, in execute_patch
    frappe.get_attr(patchmodule.split()[0] + ".execute")()
  File "/Users/rohitwaghchaure/Documents/erpnext/frappe-bench/apps/erpnext/erpnext/patches/v10_0/workflow_leave_application.py", line 15, in execute
    }).insert(ignore_permissions=True)
  File "/Users/rohitwaghchaure/Documents/erpnext/frappe-bench/apps/frappe/frappe/model/document.py", line 233, in insert
    raise e
frappe.exceptions.DuplicateEntryError: (u'Workflow State', u'Open', IntegrityError(1062, u"Duplicate entry 'Open' for key 'PRIMARY'"))